src/pages/guidelines/ibm-dot-com-style-models/overview.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: IBM.com Style Models
+description: IBM.com Style Models provide a design framework that defines the foundational and structural elements of IBM.com pages in order to establish consistency and coherency across users' journeys in a scalable way.
+tabs: ['Overview', 'Style Model List']
+---
+<PageDescription>
+
+IBM.com Style Models provide a design framework that defines the foundational and structural elements of IBM.com pages in order to establish consistency and coherency across users' journeys in a scalable way.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Our vision</AnchorLink>
+<AnchorLink>Style Model</AnchorLink>
+<AnchorLink>Using Style Models</AnchorLink>
+
+</AnchorLinks>
+
+
+## Our vision
+
+Our vision is that every visitor to IBM.com has a productive, efficient and delightful experience.
+
+
+IBM.com is a collection of highly functional, versatile, and diverse contents and experiences. It serves different audience groups and their needs, in different industries and geographic regions, and covers crucial portion of our users' journeys.
+
+
+Here are the three principles that we set up to reach our vision of helping our users in the best way possible.
+
+<Title>The experience is the product</Title>
+
+The user's experience along the journey—spanning from the user's initial research through decision making, purchase, management, and all the way to the actual use of the given product—is becoming a core aspect of any product. 
```

Comment:
Suggestion to number the principles, might be nice for the user to follow.

Reply:
Thank you, added numbers.